THIRD_PARTY.md

```diff
@@ -28,6 +28,40 @@ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 ```
 
+## libmve
+
+libmve - Interplay MVE implementation from D2X project. https://github.com/btb/d2x
+
+[@c030c453](https://github.com/btb/d2x/tree/c030c4531ad19f1658ea9635ff4ee6861e1d15e0)
+
+* libmve/decoder8.c
```

Comment:
they are now .cpp files. It is not clear which files are taken from D2X as-is, and which ones have been changed for the needs of D3. So technically it's not a third-party that we can update easily simply by pulling an upstream commit, because some files are ours and some are not

Reply:
I'd like to keep information about authors somewhere to honor their efforts. Third-party is best place for this.